website/docs/d/third_party_integrations.markdown

```diff
@@ -77,5 +77,13 @@ Additional values based on Type
 * `WEBHOOK`
    * `url` - Your webhook URL.
    * `secret` - An optional field for your webhook secret.
+* `MICROSOFT_TEAMS`
```

Comment:
we should also add an example of how to use this resource and data source in [terraform-provider-mongodbatlas/tree/master/examples](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/master/examples)

Reply:
example added

Reply:
FYI we have not added an example for everything as it is not always worth the time cost.  We should be more thoughtful on what needs an example or not imo @andreaangiolillo.   I'll define this going forward in the tickets.